app/src/main/res/values/strings.xml

```diff
@@ -33,6 +33,8 @@
     <string name="reorder_by_dragging">Reorder folders by dragging</string>
     <string name="reorder_by_dragging_pro">Reorder folders by dragging (Pro)</string>
     <string name="restore_to_path">Restoring to \'%s\'</string>
+    <string name="restore_all_selected_confirmation">Are you sure you want to restore all selected files?</string>
+    <string name="restore_all_confirmation">Are you sure you want to restore all files?</string>
```

Comment:
I think we can merge these into a plural string that supports `one` and `other` quantities. Example final string:

```
Are you sure you want to restore x file(s)?
```